docs/source/concepts/package-naming-conv.rst

```diff
@@ -48,39 +48,49 @@ to parse it for any specific information.
     pair: terminology; canonical name
     seealso: name; canonical name
 
-canonical name
+Canonical name
 ==============
 
-The package name, version and build string joined together by
+The package name, version, and build string joined together by
 hyphens---name-version-buildstring. In usage documentation, these
 are referred to by ``canonical_name``.
 
 .. _filename:
 .. index::
     pair: terminology; filename
 
-filename
+Filename
 ========
 
 Conda package filenames are canonical names, plus the suffix
-``.tar.bz2``.
+``.tar.bz2`` or ``.conda``.
 
 The following figure compares a canonical name to a filename:
 
 .. figure:: /img/conda_names.png
```

Comment:
This image doesn't seem to come up in the rendered docs http://conda.github.io/conda-build/pr-3674/concepts/package-naming-conv.html